postmarketOS/pmbootstrap
# qemu-amd64: make xfce4, mate work out of the box with SDL display

Fixes #672.
* `mate`, `xfce4`: depend on `mesa-egl`
* `xfce4`: update config, so it does not have the hardcoded `/home/user` reference anymore (https://github.com/postmarketOS/xfce4-phone/pull/2)

### How to test

This should start QEMU without problems:
```shell
pmbootstrap config ui xfce4 && \
  pmbootstrap -y zap && \
  yes | pmbootstrap -y install --no-fde && \
  pmbootstrap qemu
```
